tests/server_tests.py

```diff
@@ -159,6 +160,13 @@ def __init__(self, port, smtp_port):
             '--datastore_consistency_policy=consistent',
         ])
 
+    def flush_output(self):
+      """Flushes the buffered output from this subprocess to stderr."""
+      self.output, lines_to_print = [], self.output
+      if lines_to_print:
+        if self.BIND_RE.search('\n'.join(lines_to_print)):
+            sys.stderr.write('%s failed port 8081 already in use.\n' % self.name)
```

Comment:
Can you also suggest to turn down the local Person Finder server or the server test if any, in the error message? It would help users like you to understand the reason of the error :)

Also, can you extract the error message "port 8081 already in use" from lines_to_print instead of hard-coding it? It might be possible that it outputs BindError for other ports, and this message would be confusing in that case.